opencensus/exporters/stats/stackdriver/internal/stackdriver_exporter.cc

```diff
@@ -107,6 +108,8 @@ void Handler::ExportViewData(
     for (int i = rpc_index * kTimeSeriesBatchSize; i < batch_end; ++i) {
       *request.add_time_series() = time_series[i];
     };
+    responses[rpc_index].second.set_deadline(
```

Comment:
This responses name is definitely not what I was expecting to be. Consider to change to a more relevant name.

Reply:
Yeah, I was surprised also. I'll change it in a followup PR.